--- src/mod_variety.c	2003-03-19 21:02:44.000000000 +0100
+++ src/mod_variety.c.oden	2003-03-24 20:49:23.000000000 +0100
@@ -299,8 +299,18 @@
     c->directory_max = atoi(w);
     return NULL;
 }
+
+/* ######################################################################## */
+static int variety_init_handler(apr_pool_t *p, apr_pool_t *plog,
+                                apr_pool_t *ptemp, server_rec *s)
+{
+    ap_add_version_component(p, "mod_variety/0.2.0");
+    return OK;
+}
+
 /* ######################################################################## */
 static void variety_register_hooks (apr_pool_t *p) {
+    ap_hook_post_config(variety_init_handler, NULL, NULL, APR_HOOK_MIDDLE);
     ap_hook_handler(variety_redirect_handler, NULL, NULL, APR_HOOK_MIDDLE);
     ap_hook_fixups(variety_fixups, NULL, NULL, APR_HOOK_FIRST);
 
